README.md

```diff
@@ -1,4 +1,4 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/853yby19lvrrd435/branch/master?svg=true)](https://ci.appveyor.com/project/stryker-mutator/stryker-net/branch/master)
+[![Build Status](https://dev.azure.com/stryker-mutator/Stryker/_apis/build/status/stryker-net)](https://dev.azure.com/stryker-mutator/Stryker/_build/latest?definitionId=4)
```

Comment:
I think it will. Else we could always specify that we want the badge for the master branch, that should also work but I wanted to try it like this first.